content/docs/04-clusters/03-edge/04-edgeforge-workflow/12-build-artifacts.md

```diff
@@ -0,0 +1,309 @@
+
+</WarningBox>
+<br />
+
+In this guide, you will begin with installing the necessary tool, the Palette Edge CLI, on your development machine. Next, you will create a cluster profile in Palette referencing one of the provider images you built using the [Build Edge Artifacts](/clusters/edge/edgeforge-workflow/palette-canvos) workflow. Then, you will access the cluster profile using the Palette Edge CLI utility to create a content bundle. Lastly, when your content bundle is ready, you will use the CanvOS utility to embed the content bundle, site-specific configuration, and user data into the Edge installer ISO image.  
+  
+
+The diagram below displays the high-level steps to build the Edge installer ISO image using a content bundle.
+
+![An overarching diagram displaying the workflow in the current guide.](/clusters_edge_edge-forge-workflow_build-images_build-artifacts_overarching.png)
+
+
+# Prerequisites
+
+To complete this how-to guide, you will need the following items:
+<br />
+
+* All prerequisites outlined in the [Build Edge Artifacts](/clusters/edge/edgeforge-workflow/palette-canvos) guide, except for this guide, your Linux machine must have 100 GB of storage or higher. The actual storage will depend on the size of the content bundle you will use to build the Edge installer ISO image.
+
+
+* Provider images are built following the [Build Edge Artifacts](/clusters/edge/edgeforge-workflow/palette-canvos) guide.
+
+
+* One of the provider images is pushed to an image registry. This guide uses a provider image compatible with K3s v1.25.x pushed to the [ttl.sh](https://ttl.sh/) image registry. However, you can use any other provider image and registry per your requirements.
+
+
+# Instructions
+
+Use the following instructions on your Linux machine, which this guide refers to as the development environment.
+<br />
+
+1. The first few steps will guide you to download, install, and verify the Palette Edge CLI. The Palette Edge CLI is a command-line utility to interact with Palette and perform specific tasks in your development environment, such as creating a content bundle. 
+
+  Issue the following command to download the Palette Edge CLI. This command uses `curl` to download the Palette Edge CLI binary from the specified URL and save it as **palette-edge** in the current directory. 
+<br />
+
+  ```bash
+  curl https://software.spectrocloud.com/stylus/v3.4.3/cli/linux/palette-edge --output palette-edge
+  ```
+
+
+2. Set the executable permissions for the **palette-edge** binary by issuing the following command.
+<br />
+
+  ```bash
+  chmod 755 palette-edge
+  ```
+
+
+3. Use the following command to move the **palette-edge** binary to the **/bin/** directory. You will need elevated privileges to perform this operation. This step will allow you to issue the `palette-edge` command from any directory in your development environment.
```

Comment:
```suggestion
3. Use the following command to move the palette-edge binary to the **/usr/local/bin** directory so that the binary is available in your system $PATH.  This will allow you to issue the `palette-edge` command from any directory in your development environment.
```

Use `/usr/local/bin` instead of `/bin`. `/bin` is often a protected file folder so removing binaries is often prohibited. This is the case for Mac environments, the preferred approach is for `/usr/local/bin` as the binary is scoped to the current users. Now, if you know for sure that the binary needs to be globally scoped then use `/bin` but do so with care.